readme.md

```diff
@@ -518,7 +518,7 @@
 - [Markdown](https://github.com/BubuAnabelas/awesome-markdown) - Markup language.
 - [Dev Fun](https://github.com/mislavcimpersak/awesome-dev-fun) - Funny developer projects.
 - [Events in the Netherlands](https://github.com/awkward/awesome-netherlands-events) - Tech-related events in the Netherlands.
-
+- [Esoteric](https://github.com/angrykoala/awesome-esoteric) - Esoteric programming languages.
```

Comment:
I think the title should be: `Esoteric Programming Languages`. Just `Esoteric` is too ambiguous. Same in your readme.

> Include a short description about the list project/theme. **It should not describe the list itself.**
Example: `- [Fish](…) - User-friendly shell.`

Maybe:

> Programming languages designed for experimentation and jokes rather than actual use.